MediaBrowser.Providers/Manager/ItemImageProvider.cs

```diff
@@ -58,9 +58,9 @@ public bool ValidateImages(BaseItem item, IEnumerable<IImageProvider> providers,
 
             if (!(item is Photo))
             {
-                var images = providers.OfType<ILocalImageProvider>()
-                    .SelectMany(i => i.GetImages(item, directoryService))
-                    .ToList();
+                var images = providers
+                                .OfType<ILocalImageProvider>()
+                                .SelectMany(i => i.GetImages(item, directoryService));
```

Comment:
I believe the indentation of four spaces is standard in this codebase.

Reply:
cc @Bond-009

Reply:
It is

Reply:
Understood, will fix.